### PR TITLE
show the activity streams even if the current user has not submitted any solutions

### PR DESCRIPTION
**Pull request by MohanL**
This PR fixes https://github.com/exercism/exercism.io/issues/3220.

We have a check of the user's submissions before we show the activity streams page. I removed the check

### CODE REVIEW

**Comment by kytrinyx** [comment]
Good call. One thing to consider: what do you see if you click on the activity stream and you haven't done anything yet? Could you provide a screenshot of this scenario?

**Comment by MohanL** [comment]
@kytrinyx 
that will be solved by this PR https://github.com/exercism/exercism.io/pull/3223

File: app/views/application/_navbar.erb [hunk 1]
```diff
@@ -22,10 +22,8 @@
       </ul>
 
       <ul class="nav navbar-nav navbar-right">
-          <% if !current_user.guest? %>
-            <% if current_user.sees_exercises? %>
-              <li><a href="/inbox">Activity Streams</a></li>
-            <% end %>
+        <% if !current_user.guest? %>
```
Comment (mhelmetag) [inline app/views/application/_navbar.erb]: Can we do `unless` vs `if !`?

Comment (MohanL) [inline app/views/application/_navbar.erb]: Good catch, I should use unless here 💯

**Comment by kytrinyx** [comment]
Yeah, having read through the various related PRs and the discussions, I believe that this is _not_ the solution we want.

We don't want to link to the activity stream in the nav if you haven't submitted anything, we simply want to show you something useful if you happen to go to it (because someone shared a link, for example).

I'm going to close this PR. Thanks so much @MohanL for working on this—I don't think we'd find the right solution if we didn't have these to start the conversation.

**Pull request closed without merging**